jest.config.base.js

```diff
@@ -187,8 +180,9 @@ module.exports = {
   //   "/node_modules/",
   //   "\\.pnp\\.[^\\/]+$"
   // ],
-    transformIgnorePatterns: ['node_modules/(?!(@mapbox/tiny-sdf|mapbox-gl)/)'],
-
+  transformIgnorePatterns: [
+    'node_modules/(?!(@mapbox/tiny-sdf|mapbox-gl|d3-array|internmap|d3-color|d3-scale|d3-interpolate|d3-format|d3-time|d3-time-format)/)'
```

Comment:
At the end this was enough. I thought I had already tried well with all d3 deps, but I was missing some